cartographer/mapping/proto/sparse_pose_graph.proto

```diff
@@ -44,8 +44,9 @@ message SparsePoseGraph {
     // Pose of the scan relative to submap, i.e. taking data from the scan frame
     // into the submap frame.
     optional transform.proto.Rigid3d relative_pose = 3;
-    // 6x6 square root inverse of the covariance matrix.
-    repeated double sqrt_Lambda = 4;  // NOLINT
+    // 6x6 square root inverse of the covariance matrix. This is stored in
+    // column-major order for ease of Eigen::Map.
```

Comment:
Shouldn't the covariance be a symmetric matrix, hence the square root inverse as well, i.e. be in column-major and row-major order at the same time?